step.yml

```diff
@@ -74,6 +74,7 @@ inputs:
       - "ad-hoc"
```

Comment:
The go dependency install is still there. Could you please remove it?
https://github.com/bitrise-steplib/steps-xcode-archive/blob/d21f1bc5af266f53690dd7d6e3958af487c18d6b/step.yml#L28